components/core/src/clp/CurlDownloadHandler.hpp

```diff
@@ -53,6 +54,7 @@ class CurlDownloadHandler {
      * Doc: https://curl.se/libcurl/c/CURLOPT_CONNECTTIMEOUT.html
      * @param overall_timeout Maximum time that the transfer may take. Note that this includes
      * `connection_timeout`. Doc: https://curl.se/libcurl/c/CURLOPT_TIMEOUT.html
+     * @param custom_headers Custom request headers passed by users.
```

Comment:
```suggestion
     * @param http_header_kv_pairs Key-value pairs representing HTTP headers to pass to the server
     * in the download request. Doc: https://curl.se/libcurl/c/CURLOPT_HTTPHEADER.html
```
For naming, how about `http_header_kv_pairs` instead of `custom_headers`?

Reply:
Are we planning to apply this doc string and rename variables?